src/style.css

```diff
@@ -0,0 +1,91 @@
+@font-face {
```

Comment:
add bold font as well

Reply:
It means you should add an additional `@font-face` and set `Avenir-Heavy` in src (and `bold` in font-weight as you do here)